ru/intro/conventions.rst

```diff
@@ -79,54 +79,54 @@ cледовательно ``/article-categories/view-all`` это коррект
 ``article_categories``, и ``user_favorite_pages`` полностью отвечают требованиям
 модели.
 
-Для соответствия соглашениям при именовании таблиц и их полей должны
-использоваться английские слова. В противном случае CakePHP может быть не
-способен правильно обработать словоформы (соотнесение форм единственного и
+В соответствии с соглашениями, при именовании таблиц и их полей должны
+использоваться английские слова. В противном случае CakePHP возможно не сможет
+правильно обработать словоформы (соотнесение форм единственного и
 множественного числа). Если же Вы хотите добавить правила обработки для
-некоторых слов Вашего собственного языка, Вы можете воспользоваться служебным
+некоторых слов Вашего языка, Вы можете воспользоваться служебным
 классом :php:class:`Cake\\Utility\\Inflector`.
 Помимо определения этих пользовательских служебных правил, этот класс также
 позволяет Вам проверять, что CakePHP понимает Ваш пользовательский синтаксис для
 форм единственного и множественного числа. Смотрите документацию
-:doc:`/core-libraries/inflector` для большей информации.
+:doc:`/core-libraries/inflector` для более подробной информации.
 
 Имена полей, состоящие из двух и более слов, разделяются подчеркиваниями.
 
-Внешние ключи в связях hasMany, belongsTo/hasOne распознаются по умолчанию как
-форма единственного числа имени связанной таблицы с последующим суффиксом
+Внешние ключи в связях hasMany (один ко многим), belongsTo/hasOne (один к одному) распознаются по умолчанию как
+форма единственного числа имени связанной таблицы с последующим постфиксом
 ``_id``. Таким образом если таблица Users имеет связь hasMany с таблицей
-Articles, таблица ``articles`` будет ссылаться на таблицу ``users`` по полю
-внешнего ключа ``user_id``. Для таких таблиц, как ``article_categories``, чьи
-названия состоят из нескольких слов, внешним ключом будет поле
+Articles, то таблица ``articles`` будет ссылаться на таблицу ``users`` по полю
+внешнего ключа ``user_id``. Для таких таблиц, как ``article_categories``, у
+которых названия состоят из нескольких слов, внешним ключом будет поле
 ``article_category_id``.
 
-Объединенные таблицы, получающиеся при использованием связи BelongsToMany между
+Объединенные таблицы, полученные при использовании связи BelongsToMany (многие ко многим) между
 моделями, должны быть названы таким образом, чтобы имена составляющих их таблиц
-перечислялись в алфавитном порядке(``articles_tags`` но не ``tags_articles``).
+перечислялись в алфавитном порядке (``articles_tags``, а  не ``tags_articles``).
 
 В дополнение к использованию в качестве первичого ключа обычного
-автоинкрементируемого поля, Вы также можете использовать в этих целях поля
+автоинкрементного поля, Вы также можете использовать в этих целях поля
 UUID. CakePHP создаст уникальный 36-значный UUID
 (:php:meth:`Cake\\Utility\\Text::uuid()`), всякий раз когда Вы вызовете метод
 ``Table::save()``.
 
-Соглашения вида
+Соглашения Вида (Представления)
 ===============
```

Comment:
@Leo240 yes that's what I meant. Nice work on the russian translation by the way ! 👍